specification/resources/resource-manager/Microsoft.Authorization/stable/2025-03-01/policyTokens.json

```diff
@@ -315,5 +380,15 @@
       },
       "description": "The policy log info."
```

Comment:
>managementGroupId [](http://example.com/codeflow?start=15&length=17)

Minor: MGs are actually named by the client and not guaranteed unique (across tenants, that is). I recommend calling this managementGroupName.